tools/windows-node-installer/pkg/types/types.go

```diff
@@ -157,6 +157,9 @@ func (w *Windows) Reinitialize() error {
 	if err := w.GetSSHClient(); err != nil {
 		return fmt.Errorf("failed to reinitialize ssh client: %v", err)
 	}
+	if err := w.SetupWinRMClient(); err != nil {
```

Comment:
What happens to existing connection? Shouldn't we close it?

Reply:
We do not need to close it. [NewClient()](https://godoc.org/github.com/masterzen/winrm#NewClient) that we use to initialize a client does not create a connection. A connection is created and closed with every `Run()` call.

Reply:
Got it, thanks :)